lib/install/validate/validator.go

```diff
@@ -310,6 +317,9 @@ func (v *Validator) network(ctx context.Context, input *data.Data, conf *metadat
 			},
 		},
 	})
+	if input.BridgeNetworkName == input.ManagementNetworkName {
+		v.NoteIssue(errors.Errorf("bridge network %s is used for management network, please choose different network", input.BridgeNetworkName))
```

Comment:
the bridge network must not be shared with another network role - management also uses %s